.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,24 @@
+name: Continuous Integration
+on:
+  push:
+    paths-ignore:
+      - 'todo_app/diagrams/**'
+  pull_request:
+    paths-ignore:
+      - 'todo_app/diagrams/**'
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build Docker Image do-me:test
+      run: docker build --target test --tag do-me:test .
+
+    - name: Launch Docker container do-me:test and run unit tests
+      run: docker run -e TRELLO_KEY=FAKE_KEY -e TRELLO_TOKEN=FAKE_TOKEN -e TRELLO_BOARD_NAME=ToDoApp -i do-me:test todo_app/tests
```

Comment:
Is there any reason you specify this with the `-i` flag? It's for ensuring you're able to interact with the container's stdin, but we don't expect any interaction here
Also, since you have got your `.env.test` file checked in, it would be slightly nicer to pass that through than passing each variable individually - less effort to update it if they change in future! i.e. `--env-file .env.test` rather than `-e TRELLO_KEY=<blah> -e TRELLO_TOKEN=...`